tasks/10_remove-whitespaces/remove-whitespaces.js

```diff
@@ -0,0 +1,4 @@
+export function trimWhiteSpaces(str){
+    let res = str.split(" ").join('')
+    return res
+}
```

Comment:
Please insert an empty line at the of each file. It is a good practice for programming in general